wavelink/tracks.py

```diff
@@ -210,18 +210,18 @@ def isrc(self) -> str | None:
     def source(self) -> str:
         """Property returning the source of this track as a ``str``.
 
-        E.g. "spotify" or "youtube"
+        E.g. "spotify" or "youtube".
         """
         return self._source
 
     @property
     def album(self) -> Album:
-        """Property returning album data for this track. See Also: :class:`wavelink.Album`."""
+        """Property returning album data for this track."""
         return self._album
 
     @property
     def artist(self) -> Artist:
-        """Property returning artist data for this track. See Also: :class:`wavelink.Artist`."""
```

Comment:
You can check here:
https://wavelink.dev/en/feature-v3/wavelink.html#wavelink.Playable.artist

Reference was already present above the statement.

Reply:
Keep a description of what this is though. Docs don't separate properties (descriptors) from normal attributes, and it would be advisable to let people know this is a property.

Reply:
Also if it doesn't contain a doc string it won't show up in the docs

Reply:
It was showing: "Property returning artist data for this track. See Also".
 So I removed everything after period.

Reply:
That's fine